sentry-core/src/main/java/io/sentry/core/protocol/User.java

```diff
@@ -108,4 +109,46 @@ public void setOthers(Map<String, String> other) {
   public void acceptUnknownProperties(Map<String, Object> unknown) {
     this.unknown = unknown;
   }
+
+  @TestOnly
+  Map<String, Object> getUnknown() {
+    return unknown;
+  }
+
+  @Override
+  public User clone() throws CloneNotSupportedException {
```

Comment:
can we have this implementation be on `<T> where T : IUnknownPropertiesConsumer` or something? Seems like the same code

Reply:
we are `Overriding` an `Object`'s method, it's just gonna make more complicated for something straightforward